src/ServiceInsight.Tests/MessageBodyViewModelTests.cs

```diff
@@ -2,24 +2,23 @@
 {
     using System;
     using Caliburn.Micro;
+    using MessageList;
+    using MessageViewers;
+    using MessageViewers.HexViewer;
+    using MessageViewers.JsonViewer;
+    using MessageViewers.XmlViewer;
     using Microsoft.Reactive.Testing;
+    using Models;
     using NSubstitute;
     using NUnit.Framework;
     using ReactiveUI.Testing;
+    using ServiceControl;
     using ServiceInsight.Framework;
     using ServiceInsight.Framework.Events;
-    using ServiceInsight.MessageList;
-    using ServiceInsight.MessageViewers;
-    using ServiceInsight.MessageViewers.HexViewer;
-    using ServiceInsight.MessageViewers.JsonViewer;
-    using ServiceInsight.MessageViewers.XmlViewer;
-    using ServiceInsight.Models;
-    using ServiceInsight.ServiceControl;
```

Comment:
@adamralph usings inside the namespace is why we need to use the global:: root namespace. From a technical perspective, it's designed to protect the codebase from bad choices (like naming classes with the same names as base classes), but I'd rather have the code break if we do something dumb.